Release v1.4/toevoegen maatregelen (#179)
MinBZK/Algoritmekader: docs/maatregelen/uitvoeren_risicoanalyse_en_formuleren_mitigerende_maatregelen_voor_privacyrisico's.md

---
title: Uitvoeren risicoanalyse en formuleren mitigerende maatregelen voor privacyrisico's
toelichting: Bij het uitvoeren van een risicoanalyse voor privacyrisico's is het van belang dat de geïdentificeerde risico's worden vertaald naar concrete, mitigerende maatregelen om persoonsgegevens te beschermen.
vereiste:
- dpia_verplicht_bij_hoog_risico
levenscyclus:
- ontwerp
- dataverkenning-en-datapreparatie
- ontwikkelen
- verficatie-en-validatie
- implementatie
bouwblok:
- privacy-en-gegevensbescherming
rollen:
- proceseigenaar
- data-scientist
- data-engineer
- aanbieder
- privacy-officer
- security-officer
- ethicus
- beleidsmedewerker

hide:
- navigation
---

<!-- tags -->

## Maatregel

> Uitvoeren DPIA en formuleren mitigerende maatregelen. 

Reviewer: ```suggestion
Uitvoeren van een [DPIA](../instrumenten/DPIA.md) en formuleren van mitigerende maatregelen. 
```
interne link gelegd

## Toelichting

> - Verifieer of een DPIA over het werkproces is uitgevoerd dat wordt of zal worden ondersteund met een algoritme of AI-systeem.

Reviewer: ```suggestion
- Verifieer of een DPIA is uitgevoerd over het werkproces dat wordt of zal worden ondersteund met een algoritme of AI-systeem.
```

- Voer een risico analyse uit om de privacyrisico's met de inzet van algoritmes en AI-systemen in beeld te brengen en bepaal welke impact dit heeft op betrokkenen.
- Organisatorische en technische maatregelen moeten worden getroffen om persoonsgegevens bij de ontwikkeling en het gebruik van het algoritme of AI-systeem te beschermen.
- Beleg de mitigerende maatregelen bij betrokken actoren. Denk bijvoorbeeld het toekennen van de maatregelen als anonimiseren en pseudonimiseren van persoonsgegevens aan een data engineer, voordat deze kunnen worden gebruikt ten behoeve van het ontwikkelen of controleren van het algoritme of AI-systeem. 
- Monitor de voortgang op het realiseren van de maatregelen en zorg voor bewijsstuken als deze zijn gerealiseerd. Deze bewijsstukken kunnen onderdeel worden van een audit.

> - Als er een noodzaak is om na verloop van tijd meer persoonsgegevens te werken of om andere verwerkingen uit te voeren, zal opnieuw een beoordeling moeten plaatsvinden of er privacyrisico's ontstaan en hoe deze kunnen worden gemitigeerd. Gedurende de levenscyclus van het algoritme of AI-systeem moet haandacht voor blijven voor het uitvoeren van de risicoanalyse voor privacyrisico's.

Reviewer: ```suggestion
- Als er een noodzaak is om na verloop van tijd meer persoonsgegevens te verwerken of om andere verwerkingen uit te voeren, zal opnieuw een beoordeling moeten plaatsvinden of er privacyrisico's ontstaan en hoe deze kunnen worden gemitigeerd. Gedurende de levenscyclus van het algoritme of AI-systeem moet aandacht blijven voor het uitvoeren van de risicoanalyse voor privacyrisico's.
```
